packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts

```diff
@@ -54,9 +54,11 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
 
   /**
    * Whether the ComboBox auto completes. As the user is inputing text, it will be suggested potential matches from the list of options. If
-   * the combo box is expanded, this will also scroll to the suggested option, and give it a selected style. Defaults to false.
+   * the combo box is expanded, this will also scroll to the suggested option, and give it a selected style.
+   *
+   * @default "off"

```

Comment:
Default is actually 'on': https://github.com/OfficeDev/office-ui-fabric-react/blob/89bbe88b2775fb48ae04631b135bdfbbc9806a5f/packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx#L73

Reply:
Oh interesting!  I was just lazy and translated `false` to `off`, I should have checked the component.  All the more reason to update the docs, I guess.  I've amended the commit, thanks for the quick review.